packages/hub/schema/field.js

```diff
@@ -195,7 +196,7 @@ module.exports = class Field {
 
     if (pendingChange.serverProvidedValues.has(this.id) && pendingChange.serverProvidedValues.get(this.id) === value) {
```

Comment:
I think we need to replace this `===` with `isEqual` too.

Reply:
sure

Reply:
(The server-provided value comes from a default. Either default-at-create or default-at-update.)